service/src/main/resources/db/changesets/20240611_add_control_workspace_id_to_pipeline_runs.yaml

```diff
@@ -0,0 +1,12 @@
+# Add a control_workspace_id column to the pipeline_runs table and change output type to jsonb
+databaseChangeLog:
+  - changeSet:
+      id: Add a control_workspace_id column to the pipeline_runs table
+      author: mma
+      changes:
+        - addColumn:
+            tableName: pipeline_runs
+            columns:
+              - column:
+                  name: control_workspace_id
```

Comment:
if you are making this un-nullable then you'll need to get default values for existing rows

Reply:
i think that since i'm not requiring it to be not null here, I don't have to backfill / provide default values; the non-nullable requirement in the service will apply to any new writes. this does mean this isn't a backwards-compatible change but I'm ok with that since we're still just in dev.